src/Native/build.sh

```diff
@@ -98,3 +98,7 @@ set -x # turn on trace
 cmake "$DIR" -G "Unix Makefiles" $__cmake_defines
 set +x # turn off trace
 make install
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "Changing libMklImports.dylib's executable path within libSymSgdNative.dylib so that loader can find it."
+    install_name_tool "-change" @loader_path/libMklImports.dylib @rpath/libMklImports.dylib "$RootRepo"/bin/x64."$__configuration"/Native/libSymSgdNative.dylib
```

Comment:
How is this going to work on an end-user's machine, when they use our NuGet packages?

Reply:
Note: you can do this stuff inside of CMake too. It may make more sense there, since you won't have to do things like `"$RootRepo"/bin/x64."$__configuration"/Native`.  Plus then it can also be localized in the `SymSgdNative\CmakeLists.txt`, and this high-level build script doesn't need to know about individual assemblies we are building.

Here's an example:  [https://github.com/dotnet/corefx/blob/20bcc468b12119e12428cfd9cce50ef5d5b9cc13/src/Native/System.Security.Cryptography.Native/CMakeLists.txt#L56-L70](https://github.com/dotnet/corefx/blob/20bcc468b12119e12428cfd9cce50ef5d5b9cc13/src/Native/System.Security.Cryptography.Native/CMakeLists.txt#L56-L70)

---
In reply to: [206888376](https://github.com/dotnet/machinelearning/pull/624#discussion_r206888376) [](ancestors = 206888376)

Reply:
We spoke about it offline and the insall_name_tool step can moved out of the script and be done when nuget is created. I'll test the nuget as end user during bug-bash. 

---
In reply to: [206964367](https://github.com/dotnet/machinelearning/pull/624#discussion_r206964367) [](ancestors = 206964367,206888376)